src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -626,6 +625,23 @@ class LeVADocumentUseCase extends DocGenUseCase {
         return uri
     }
 
+    /**
+     * If the risk is associated to technical specification task which is itself associated with a User story,
+     * return the technical specification task
+     * otherwise, return the system requirements
+     * @param risk
+     * @return
+     */
+    private List<Project.JiraDataItem> getRequirements(Project.JiraDataItem risk) {
```

Comment:
There will never be more than one requirement, so you don't need to return a List. Just return JiraDataItem.

Reply:
We can´t send `JiraDataItem`, Because, each risk can have multiple Tech Specifications or System Requirements. So, It should be List<>